src/components/Charts/ModelChart.js

```diff
@@ -26,116 +30,140 @@ const ModelChart = ({
   interventions,
   currentIntervention,
 }) => {
-  const { isEmbed } = useEmbed();
-  const interventionToModel = {
-    [INTERVENTIONS.LIMITED_ACTION]: interventions.baseline,
-    [INTERVENTIONS.SOCIAL_DISTANCING]:
-      interventions.distancingPoorEnforcement.now,
-    [INTERVENTIONS.SHELTER_IN_PLACE]: interventions.distancing.now,
-  };
-
-  let model = interventionToModel[currentIntervention];
+  let interventionsModelIsValid = true;
+  let model;
 
-  if (currentIntervention === INTERVENTIONS.SHELTER_IN_PLACE) {
-    model = interventionToModel[INTERVENTIONS.SOCIAL_DISTANCING];
+  // In future we can do further model shape/schema checking
```

Comment:
So this file has a lot more diffs than I expected and it's hard to figure out what's changing / why.  Can you confirm these are intentional as opposed to a screwy merge or something? Thanks!

Reply:
Sorry -- this looks like more changes than it is. Let me leave a couple of comments within the file explaining what each change is.